donut/email_utils.py

```diff
@@ -10,7 +10,7 @@ def send_email(to, text, subject, use_prefix=True, group=None):
     is not none, the email is sent to a newsgroup and the to emails are hidden.
     """
 
-    msg = MIMEText(msg)
+    msg = MIMEText(text, "html")
```

Comment:
We need to escape this in case they use `<`, `>`, or `&` in emails. How about running newsgroup emails through a Markdown converter? This would allow people to send formatted emails without writing raw HTML, and would automatically take care of escapes.

Reply:
Also, can you remove the `MIMEMultipart` import as part of this PR?

Reply:
> We need to escape this in case they use `<`, `>`, or `&` in emails. How about running newsgroup emails through a Markdown converter? This would allow people to send formatted emails without writing raw HTML, and would automatically take care of escapes.

lol is it time to use showdown.js again

Reply:
I completely forgot about xss attacks lol. 
I think I'll just stick with text/plain until later.